docker-compose-local-testing.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+
+services:
+  postgres:
+    hostname: postgres
+    image: postgres:14.4-alpine
+    volumes:
+      - postgres-volume:/var/lib/postgresql/data
+    networks:
+      - codecovapi
+    ports:
+      - "5436:5432"
```

Comment:
I would prefer to have this use port 5432 and have redis use port 6379 since I think the settings use those ports as default and it would be easier for devs to avoid modifying those